doc/setup/installing_windows_python.adoc

```diff
@@ -0,0 +1,30 @@
+= Installing OSI for Python on Windows
+
+**Prerequisites**
+
+* You have installed _Python_ with administrator rights.
+* Make sure _Python_ is added to `PATH`.
+
+**Steps**
+
+. Open a terminal.
+. Clone the Open Simulation repository.
++
+[source]
+----
+git clone https://github.com/OpenSimulationInterface/open-simulation-interface.git
+----
++
+. Change into the repository directory.
++
+[source]
+----
+cd open-simulation-interface
+----
++
+. Run the setup script.
++
+[source]
+----
+python setup.py install
+----
```

Comment:
Please review @pmai

Reply:
Looks good to me, any further tuning of installation guidance can be done when we improve installation/cmake stuff.